src/sharedInterfaces/telemetry.ts

```diff
@@ -17,6 +17,7 @@ export enum TelemetryViews {
     UserSurvey = "UserSurvey",
     General = "General",
     ConnectionDialog = "ConnectionDialog",
+    QueryPlan = "QueryPlan",
```

Comment:
Query plan or execution plan? I see that the code uses a lot of interfaces contains 'ExecutionPlan'. Let's try to keep the feature name consistent. Helps in searching and refactoring stuff.